src/pages/edit_resource/EditResourcePage.tsx

```diff
@@ -257,18 +260,23 @@ const EditResourcePage: FC = () => {
   };
 
```

Comment:
Kunne lagd til en knapp til redigering av ressursene i oversikten på "mine ressurser" som linker til redigering av eksisterende ressurs.
Her er Stines figma for mine ressurser https://www.figma.com/file/G7dPPeZnEAJhvUvhEPZb2C/prototype-DLR?node-id=3%3A2592

Reply:
Kan legges inn som egen task?

Reply:
jeg så for meg den knappen som egen task, ja. Skal opprette den